readthedocs/projects/tasks.py

```diff
@@ -648,14 +649,6 @@ def setup_vcs(self):
             log.warning('There was an error with the repository', exc_info=True)
             # Re raise the exception to stop the build at this point
             raise
-        except vcs_support_utils.LockTimeout:
```

Comment:
This exception is being captures by the top level tasks (sync_repo, and build_docs)